client_dashboard/common/src/commonMain/kotlin/org/thechance/common/presentation/overview/OverviewScreen.kt

```diff
@@ -1,52 +1,243 @@
+                        }
+                    }
+                )
+            }
+        }
+    }
 
-        OverviewContent(
-            state = state,
-            onClickBack = { navigate.popUntilRoot() }
-        )
+
+    @Composable
+    override fun Content() {
+        val tabNavigator = LocalTabNavigator.current
+        val screenModel = getScreenModel<OverviewScreenModel>()
+        Init(screenModel)
+        LaunchedEffect(Unit) {
+            collectNavigationEffect(screenModel, tabNavigator)
+        }
     }
 
-}
+    private suspend fun collectNavigationEffect(
+        screenModel: OverviewScreenModel,
+        tabNavigator: TabNavigator
+    ) {
+        screenModel.effect.collect {
+            it?.let {
+                when (it) {
+                    OverviewUiEffect.ViewMoreUsers -> {
+                        tabNavigator.current = UsersTab
+                    }
+
+                    OverviewUiEffect.ViewMoreRestaurant -> {
+                        tabNavigator.current = RestaurantsTab
+                    }
 
-@OptIn(ExperimentalMaterial3Api::class)
-@Composable
-private fun OverviewContent(
-    state: OverviewUiState,
-    onClickBack: () -> Unit,
-) {
+                    OverviewUiEffect.ViewMoreTaxis -> {
+                        tabNavigator.current = TaxisTab
+                    }
+                }
 
-    Column(
-        Modifier.background(Theme.colors.surface).fillMaxSize()
+            }
+        }
+    }
+
+    @Composable
+    private fun OverviewCard(
+        title: String,
+        onLeadingButtonClicked: () -> Unit,
+        body: @Composable () -> Unit,
+        paddingValues: PaddingValues = PaddingValues(24.kms),
+        modifier: Modifier = Modifier,
     ) {
-        Box(Modifier.weight(1f)) {
-            Text(text = "Main Screen", color = Theme.colors.onPrimary)
 
+        Column(
+            modifier = modifier.background(Theme.colors.surface, RoundedCornerShape(8.kms)).padding(paddingValues),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.spacedBy(16.kms)
+        ) {
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(
+                    text = title,
+                    style = Theme.typography.headline,
+                    color = Theme.colors.contentPrimary,
+                    modifier = Modifier.weight(2f)
+                )
+
+                BpOutlinedButton(
+                    shape = RoundedCornerShape(Theme.radius.small),
+                    title = Resources.Strings.viewMore,
+                    onClick = onLeadingButtonClicked,
+                    textStyle = Theme.typography.body,
+                    textPadding = PaddingValues(0.dp),
+                    modifier = Modifier.height(32.dp).weight(1f),
```

Comment:
```suggestion
                    modifier = Modifier.height(32.kms).weight(1f),
```

Reply:
if i use kms it changes the radius for the button in the design, it work with dp